### PR TITLE
[Hacktoberfest] Copy-edited FlyteCTL docs

### PR DESCRIPTION
**Pull request by Sayanta66**
# TL;DR

The PR fixes all of the grammatical problems and other problems of one third of the docs in the general section.

## Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Plugin
- [x] Docs

## Are all requirements met?

- [ ] Code completed
- [ ] Smoke tested
- [ ] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description

I have copyedited one third of all the docs in the general section. I have fixed the punctuation errors and other grammatical errors.

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1524

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by kumare3** [comment]
@Sayanta66 can you please update the PR description?

**Comment by Sayanta66** [comment]
> @Sayanta66 can you please update the PR description?

More description is needed?

**Comment by kumare3** [comment]
@Sayanta66  this is great, just remove the instructions?

**Comment by Sayanta66** [comment]
> @Sayanta66  this is great, just remove the instructions?

Done

**Comment by kumare3** [comment]
@Sayanta66 you also need to sign off on DCO for the check to pass

**Comment by samhita-alla** [comment]
Also, @Sayanta66, can you work on:
* Make sure flags' descriptions respect capitalized case
Example: "attribute file name to be used for updating attribute for the resource type." in [this](https://docs.flyte.org/projects/flytectl/en/stable/gen/flytectl_update_cluster-resource-attribute.html) page should be "Attribute ..." 
* Getting the docs consistent with having flytectl command instead of bin/flytectl, which is used in some places
* Render code properly in [this](https://docs.flyte.org/projects/flytectl/en/stable/gen/flytectl_completion.html) page

Moreover, you have fixed the punctuations. Can you also please make sure that the content is free of any grammatical errors?

**Comment by Sayanta66** [comment]
Okay will be doing that tonight

**Comment by samhita-alla** [comment]
> Okay will be doing that tonight

Thank you! :)

**Comment by evalsocket** [comment]
@Sayanta66 There is one issue in your pr. Please fix the grammatical issues from the code `cmd/`  and then after merge GitHub action will generate the docs for you

**Comment by Sayanta66** [comment]
Okay sure will be doing that. Thanks for the input @evalsocket

**Comment by Sayanta66** [comment]
Hi @samhita-alla I will do the thing by today and Because I was sick, I couldn't gave my entire time earlier.

**Comment by Sayanta66** [comment]
@evalsocket can you please tell me again, what I should change? I am not finding it.

**Comment by evalsocket** [comment]
@Sayanta66 Yes. Every command has its own documentation. It's cobra command documentation, That we use in cli `--help`. We generate docs for the website. So if you fix the docs from every command then it will fix the issue for both cli/website.

**Comment by samhita-alla** [comment]
> > > @evalsocket can you please tell me again, what I should change? I am not finding it.
> > 
> > 
> > @Sayanta66, I think @evalsocket is trying to say that content modifications are to be done only in the `cmd` directory, but not in `gen`. Apologies for the miscommunication in the contribution guide. All the content you see should be available in `cmd`, so please initiate your modifications in there.
> 
> @samhita-alla do I have to redo the whole thing? Btw in CMD folder all are go extension files. In the doc folder there are things that needs to be rectified like the punctuation errors, grammatical errors and all

Yes, sadly, you need to. :/ The content you see in the doc/ folder is also present in the cmd/ folder. So you should find all the content you worked on in the cmd/ folder as well. For example, https://github.com/flyteorg/flytectl/blob/master/docs/source/gen/flytectl_get_cluster-resource-attribute.rst is equivalent to https://github.com/flyteorg/flytectl/blob/master/cmd/get/matchable_cluster_resource_attribute.go.

@Sayanta66, does this help you?

**Comment by Sayanta66** [comment]
Thanks for hekping me out @samhita-alla
